internal/slackclient/client.go

```diff
@@ -162,6 +186,64 @@ func (c *SlackClient) get(path string, params map[string]string) ([]byte, error)
 	return body, nil
 }
 
+func (c *SlackClient) post(path string, params map[string]string, msg *SendMessage) ([]byte, error) {
```

Comment:
Note for later: There might be an opportunity to create a common `request_helper` method that contains common code between `get` and `post` and have those methods call the `request_helper`. There's currently quite a bit of duplicate code, but I'd favour functionality over form on a fork repo during a hackathon. 😛

Reply:
Yeah I had the same thought in the previous PR 😄